docs/src/main/asciidoc/security-openid-connect-dev-services.adoc

```diff
@@ -63,8 +76,17 @@ Finally you can click a `Logged in` option if you'd like to log out and authenti
 
 [NOTE]
 ====
-You may need to register a redirect URI for the authorization code flow initiated by Dev UI for Keycloak to work.
-Select a `Keycloak Admin` option in the right top corner, login as `admin:admin`, select the test realm and the client which Dev UI for Keycloak is configured with and add `http://localhost:8080/q/dev/io.quarkus.quarkus-oidc/provider` to `Valid Redirect URIs`.
+You may need to register a redirect URI for the authorization code flow initiated by Dev UI for Keycloak to work because Keycloak may enforce that the authenticated users are redirected only to the configured redirect URI. It is recommended to do in production to avoid the users being redirected to the wrong endpoints in case which might happen if the correct `redirect_uri` parameter in the authentication request URI has been manipulated.
+
+If Keycloak does enforce it then you will see an authentication error informing you that a `redirect_uri` value is wrong.
+
+In this case select a `Keycloak Admin` option in the right top corner, login as `admin:admin`, select the test realm and the client which Dev UI for Keycloak is configured with and add `http://localhost:8080/q/dev/io.quarkus.quarkus-oidc/provider` to `Valid Redirect URIs`. If you used `-Dquarkus.http.port` when starting Quarkus then change `8080` to the value of `quarkus.http.port`.
+
+If the container is shared between multiple applications running on different ports then you will need to register `redirect_uri` values for each of these applications.
+
+You can set the `redirect_uri` value to `*` only for the test purposes, especially when the containers are shared between multiple applications.
+
+`*` `redirect_uri` value is set by `Dev Services for Keycloak` when it creates a default realm, if no custom realm is imported.
```

Comment:
I think you should start by that maybe. Stating that if you just use the default, it's configured like that?